types/tx_result.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	abci "github.com/tendermint/abci/types"
+)
+
+// TxResult contains results of executing the transaction.
+//
+// One usage is indexing transaction results.
+type TxResult struct {
+	Height    uint64                 `json:"height"`
```

Comment:
I think we've already discussed this before. `uint` because it is never negative and only goes up. `64` because we want to preserve the size (ie. 1st node has 386 arch, 2nd node amd64)